includes/class-wc-google-analytics-js.php

```diff
@@ -1,4 +1,8 @@
 <?php
+
+// @phpcs:disable Squiz.Classes.ClassFileName.NoMatch
+// @phpcs:disable Squiz.Classes.ValidClassName.NotCamelCaps
+
```

Comment:
Again same as above, preferable to configure this for all files at once.

Reply:
Fixed here https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/260/commits/302924fe1042885e81d23730afe11bcbab1eca1b